addons/payment_adyen/models/payment_transaction.py

```diff
@@ -93,6 +93,56 @@ def _send_payment_request(self):
         _logger.info("payment request response:\n%s", pprint.pformat(response_content))
         self._handle_feedback_data('adyen', response_content)
 
+    def _send_refund_request(self, refund_amount=None):
+        """ Override of payment to send a refund request to Adyen.
+
+        Note: self.ensure_one()
+
+        :return: None
+        """
+        super()._send_refund_request(refund_amount)
+        if self.provider != 'adyen':
+            return
+
+        if not refund_amount:
+            refund_amount = self.amount
+
+        refund_tx = self._create_refund_transaction(refund_amount)
+
+        converted_amount = payment_utils.to_minor_currency_units(
+            refund_amount, refund_tx.currency_id, CURRENCY_DECIMALS.get(refund_tx.currency_id.name)
+        )
+
+        data = {
+            'merchantAccount': refund_tx.acquirer_id.adyen_merchant_account,
+            'amount': {
+                'value': converted_amount,
+                'currency': refund_tx.currency_id.name,
+            },
+            'reference': refund_tx.reference,
+        }
+        response_content = refund_tx.acquirer_id._adyen_make_request(
+            url_field_name='adyen_checkout_api_url',
+            endpoint='/payments/{}/refunds',
+            source_reference=refund_tx.acquirer_reference,
+            payload=data,
+            method='POST'
+        )
+        _logger.info("refund request response:\n%s", pprint.pformat(response_content))
+
+        # Handle the refund request response
+        pspreference = response_content.get('pspReference')
+        status = response_content.get('status')
+        if pspreference and status == '[received]':
+            # the PSP reference associated with this /refund request is different from the
+            # psp reference associated with the original payment request
+            refund_tx.acquirer_reference = pspreference
```

Comment:
Does it make sense to create the refund tx with an incorrect acquirer reference then?